pkg/aws/testutil_test.go

```diff
@@ -62,24 +68,51 @@ const (
 	fakeRDSSnapshotName = "rds-snapshot"
 )
 
-func fakeReportItemDeleting() *clusterservice.ReportItem {
+func fakeReportItemDeleted(arn string) *clusterservice.ReportItem {
+	return &clusterservice.ReportItem{
+		ID:           arn,
+		Name:         fakeResourceIdentifier,
+		Action:       clusterservice.ActionDelete,
+		ActionStatus: clusterservice.ActionStatusComplete,
+	}
+}
+
+func fakeReportItemDeleting(arn string) *clusterservice.ReportItem {
 	return &clusterservice.ReportItem{
-		ID:           fakeRDSClientInstanceARN,
+		ID:           arn,
 		Name:         fakeRDSClientInstanceIdentifier,
 		Action:       clusterservice.ActionDelete,
 		ActionStatus: clusterservice.ActionStatusInProgress,
 	}
 }
 
-func fakeReportItemDryRun() *clusterservice.ReportItem {
+func fakeReportItemDryRun(arn string) *clusterservice.ReportItem {
 	return &clusterservice.ReportItem{
-		ID:           fakeRDSClientInstanceARN,
+		ID:           arn,
 		Name:         fakeRDSClientInstanceIdentifier,
 		Action:       clusterservice.ActionDelete,
 		ActionStatus: clusterservice.ActionStatusDryRun,
 	}
 }
```

Comment:
I think all these fake reports can be refactored to take a modifyFn, like the example below

Reply:
:+1: